roles/edpm_bootstrap/tasks/bootstrap.yml

```diff
@@ -126,8 +126,6 @@
 - name: Configure swap
   ansible.builtin.import_tasks: swap.yml
 
-- name: Set GID hugetlbfs user to match kolla pre-set
-  ansible.builtin.group:
-    name: hugetlbfs
-    gid: 42477
-  become: true
+- name: Configure host users and groups
+  ansible.builtin.import_role:
+        name:  edpm_users
```

Comment:
Can we just call the role in the `configure_network` playbook just after `edpm_bootstrap` role invocation?